charts/latest/azurefile-csi-driver/templates/csi-azurefile-controller.yaml

```diff
@@ -193,6 +193,16 @@ spec:
             timeoutSeconds: 10
             periodSeconds: 30
           env:
+{{- if and .Values.workloadIdentity.clientID .Values.workloadIdentity.tenantID}}
```

Comment:
We won't rely on Azure Workload Identity Webhook to inject the environment variables

Reply:
:+1:

Reply:
:+1: We won't rely on the webhook either because we [set the env vars on the azurefile deployment as read from a secret](https://github.com/openshift/azure-file-csi-driver-operator/blob/994c32c927a69df33f83aed440cc150e8d270341/assets/controller.yaml#L54-L64). I think this will align with our implementation since we can just extend this to set `AZURE_FEDERATED_TOKEN_FILE` & `AZURE_TENANT_ID` which will mean that the workload identity extension will be used instead of service principal.